process-engine/process_engine_runtime
# 👷🐛 Fix Slack-Report generation after Jenkins builds

**Changes:**

1. Add `@NonCPS` decorator to `slack_send_summary` to fix the sending of the Slack-Report.
    - See this issue: https://stackoverflow.com/questions/40454558/jenkins-pipeline-java-io-notserializableexception-java-util-regex-matcher-error
2. Prevent new versions from being published after a failed build.

PR: #248

## How can others test the changes?

- Run some Jenkins-Builds.
- A full test log should now be send to slack, in addition to the report.

![bildschirmfoto 2019-02-15 um 10 01 26](https://user-images.githubusercontent.com/15343316/52845970-b18fc700-3108-11e9-9778-975090c03ce1.png)


## PR-Checklist

Please check the boxes in this list after submitting your PR:
